tests/e2e/forms/immunization-visit-form.standard-wdio-spec.js

```diff
@@ -0,0 +1,166 @@
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.BCG_VACCINE, 'yes');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.CHOLERA_VACCINE, 'CH');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.HEPATITIS_A_VACCINE, 'HA');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.HEPATITIS_B_VACCINE, 'yes');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.HPV_VACCINE, 'HPV');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.FLU_VACCINE, 'yes');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.JAP_ENCEPHALITIS_VACCINE, 'yes');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.MENINGOCOCCAL_VACCINE, 'MN');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.MMR_VACCINE, 'MMR');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.MMRV_VACCINE, 'MMRV');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.POLIO_VACCINE, 'PV');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.PENTAVALENT_VACCINE, 'DT');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.DPT_BOOSTER_VACCINE, 'DPT');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.PNEUMOCOCCAL_VACCINE, 'PCV');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.ROTAVIRUS_VACCINE, 'RV');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.TYPHOID_VACCINE, 'TY');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.VITAMIN_A_VACCINE, 'yes');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.YELLOW_FEVER_VACCINE, 'yes');
+        await genericForm.nextPage();
+        await immunizationVisitForm.addNotes();
+        await genericForm.nextPage();
+
+        expect(await immunizationVisitForm.getPatientNameSummaryPage()).to.equal(babyName);
+        expect(await immunizationVisitForm.getAppliedVaccines()).to.equal(await immunizationVisitForm.getAppliedVaccinesSummary() - 1);
```

Comment:
Yeah, these enketo pages are harder. I think it's better in the `wdio.page file`.

Adding `:not(:last-child)` worked for me: 
`$('label.question.note.or-branch.non-select.or-appearance-li:not(:last-child)')`
<img width="991" alt="Screen Shot 2022-08-05 at 12 30 31 pm" src="https://user-images.githubusercontent.com/66472237/183008304-2abf52ac-5ff9-4c28-a405-cf6781728cc1.png">

However you will need to add a comment why excluding the last child since it's not straight forward to understand.

Reply:
That worked perfectly, thank you!